packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -116,6 +116,14 @@ const getRuntimeDisplayName = (
   return schema?.display_name;
 };
 
+const getRuntimeTypeFromSchema = (
+  schemas: { name: string; runtime_type: string }[] | undefined,
+  runtime: string | undefined
+): string | undefined => {
+  const schema = schemas?.find(s => s.name === runtime);
+  return schema?.runtime_type;
+};
+
```

Comment:
Ok, thanks, I'll look into moving it there and adding an import (or whatever the syntax).

Reply:
In talking more with @ajbozarth it looks like this will remain where it is.  If moved, we'll want to do the same for the adjacent `getRuntimeDisplayName()` method. The difference between these methods and those (similarly-named) methods in `PipelineService` is that these operate from the schemas themselves, whereas the `PipelineService` methods operate off the schema _instances_ (i.e., runtime configuration instances) and is one reason I added the `FromSchema` suffix in order to distinguish the source of the value more clearly.